src/MusicStore/Controllers/StoreController.cs

```diff
@@ -50,22 +54,34 @@ public StoreController(MusicStoreContext dbContext)
         {
             var cacheKey = string.Format("album_{0}", id);
             Album album;
-            if (!cache.TryGetValue(cacheKey, out album))
+
+            if (AppSettings.Value.CacheTimeout > 0)
+            {
+                if (!cache.TryGetValue(cacheKey, out album))
+                {
+                    album = await DbContext.Albums
```

Comment:
This is code duplication. Why not just check if 
`if(cacheTimeout > 0 && cache.TryGetValue(...) `